### DISCUSSION TITLE
Deployments NextJS Application failed, but doesnt have a logs

### DISCUSSION THREAD

**Discussion by mahhung12** [vercel/community#4702] [Help] [answered]
### Summary

I am setting up a nextjs project and have run build success locally, but when I deploying to vercel, the status is always error and no log is displayed.

Can someone help me find a solution?

![image](https://github.com/vercel/community/assets/81293286/f2ef6563-b282-4aca-b6d3-23fd72321b33)
![image](https://github.com/vercel/community/assets/81293286/387e987c-e55f-4602-a3ac-ca6ccf7583c1)

### Steps to Reproduce

- Set up NextJS app with repository on Github
- Connect NextJS Repo with vercel
- Deployment to vercel
- Deployment fail and no log is displayed

**Comment by amyegan** [comment, marked as answer]
Hey @mahhung12. Try building on your local computer with Vercel CLI. That should provide more info about what is going wrong. You can add the `--debug` option if extra detail is needed.

https://vercel.com/docs/cli/build
